week-4/Homework/mandatory/4-groceries.js

```diff
@@ -30,18 +30,29 @@ Exercise 1:
 // Gather all week item names into this array
 let weeklyGroceriesToBuy = [];
 
-
+for (let day in weeklyMealPlan) {
+  weeklyMealPlan[day].forEach((ingredients) => {
+    weeklyGroceriesToBuy.push(ingredients);
+  });
+}
+console.log(weeklyGroceriesToBuy);
 
 /*
 Exercise 2:
   Loop through your list again, but now only collect the weekend items into the weeklyGroceriesToBuy array.
   Then use console.log() to print out the list.
 */
 // Gather weekend item names into this array
-let weekendGroceriesToBuy = [];
-
-
 
+weekendGroceriesToBuy = [];
+for (let day in weeklyMealPlan) {
+  if (day == "saturday" || day == "sunday") {
+    weeklyMealPlan[day].forEach((ingredients) => {
+      weekendGroceriesToBuy.push(ingredients);
+    });
+  }
+}
+console.log(weekendGroceriesToBuy);
```

Comment:
There is something missing here before the weekendGroceriesToBuy array which is an array of weekend = ["Saturday"," Sunday"]. Work here again.

Reply:
I think I forgot the keyword let. Other than that the code seems to work fine. Sunday does not have any items, but if we add an item ["pizza"] for example it shows in the array after Saturday items. I am missing something else here? 
Thank you for reviewing the work :)

Reply:
It works fine, It's correct Leida